### PR TITLE
refactor(demo): use a single bpmn container in the hacktoberfest demo

### LINKED ISSUE #540: [REFACTOR] Use a single BPMN container in the "hacktoberfest" demo
Use a singe HTML container and a single `BpmnVisualization` instance.

Currently, we have 1 for the default style, and for each year, 1 for dark theme and 1 for light theme.
We provide theme for 3 years (2020-2022), so we have 7 instances. We could switch to a single instance with live refresh of the style as we do in the "diagram navigation" demo provided by the TS library.

This is the same request as in #500 and #526

### PR DESCRIPTION
**Pull request by csouchet**
**Live environment of examples**

- https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/bf04dca/demo/hacktoberfest-custom-themes/index.html
- https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/1691300/demo/hacktoberfest-custom-themes/index.html
- https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/540-single_BPMN_container_hacktoberfest_demo/examples/index.html

Closes #540

### CODE REVIEW

File: demo/hacktoberfest-custom-themes/index.html [hunk 1]
```diff
@@ -122,27 +122,10 @@ <h5>Theme Year</h5>
...
-                    <h4 id="2020-light-title" class="d-hide">Light Hacktoberfest 2020 Theme</h4>
-                    <h4 id="2020-dark-title" class="d-hide">Dark Hacktoberfest 2020 Theme</h4>
-                    <h4 id="2021-light-title" class="d-hide">Light Hacktoberfest 2021 Theme</h4>
-                    <h4 id="2021-dark-title" class="d-hide">Dark Hacktoberfest 2021 Theme</h4>
-                    <h4 id="2022-light-title" class="hacktoberfest-2022-2023 d-hide">Light Hacktoberfest 2022 Theme</h4>
-                    <h4 id="2022-dark-title" class="hacktoberfest-2022-2023 d-hide">Dark Hacktoberfest 2022 Theme</h4>
-                    <h4 id="2023-light-title" class="hacktoberfest-2022-2023 d-hide">Light Hacktoberfest 2023 Theme</h4>
-                    <h4 id="2023-dark-title" class="hacktoberfest-2022-2023 d-hide">Dark Hacktoberfest 2023 Theme</h4>
-                    <h4 id="default-title" class="d-hide">Default Theme</h4>
+                    <h4 id="hacktoberfest-title"></h4>
```
Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/index.html:125]: **nitpick:** behaviour changes. The `hacktoberfest-2022-202` class is no more added for 2022 and 2023. 

before | now
----- | -----
image | image

File: demo/hacktoberfest-custom-themes/js/theme.js [hunk 2]
```diff
@@ -253,7 +253,6 @@ themes.set("2022", new Map([
             stroke: colors2022.secondaryPsybeam,
         },
         exclusiveGateway: {
-            fill: colors2022.secondaryManga,
```
Comment (tbouffard) [inline demo/hacktoberfest-custom-themes/js/theme.js:256]: **question:** why removing this? here we are supposed to only change technical things, not configuration

**Review by tbouffard — CHANGES_REQUESTED** [review]
ℹ️ I had limited time to review this PR. I was surprised it was implemented as it was not in the initial list of issues of this Sprint. So I only check the behavior
✔️ year and theme flavor change applied to the diagram
✔️ project name change correctly applied to the diagram
❌ the font of the title is no more applied for the 2022 and 2023 theme

File: demo/hacktoberfest-custom-themes/js/theme.js (hunk 2 above)
Comment (csouchet) [reply demo/hacktoberfest-custom-themes/js/theme.js:256]: Because the property `secondaryManga` doesn't exist 😆

Comment (tbouffard) [reply demo/hacktoberfest-custom-themes/js/theme.js:256]: Fair enough 😜

**Review by tbouffard — APPROVED** [review]
LGTM
✔️ Tested with the statically.io preview environment: https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/bf04dca/demo/hacktoberfest-custom-themes/index.html
ℹ️ The titles now always use the 2022-2023 font. We agreed with @csouchet it was fine

image

**Pull request merged**